tests/4.5/taskloop/test_taskloop_num_tasks.c

```diff
@@ -59,11 +59,14 @@ int test_taskloop_num_tasks() {
       }
    }
 
-   //if all the tasks in a group are run by a same thread, get TRUE else FALSE
-   OMPVV_TEST_AND_SET_VERBOSE(errors, (isGroupIdsSame(thread_ids) != 1));
+   
 
    //To check if the num_threads actually executing the parallel region > 1
-   OMPVV_TEST_AND_SET_VERBOSE(errors, (num_threads_in_parallel_zone < 1));
+   if(num_threads == 1) 
+     OMPVV_INFOMSG("Only a single thread executed the parallel region");
```

Comment:
Hi there,
FYI when looking at our CI, I see `test_taskloop_num_tasks.c` test fails compilation (with AOMP as well as with vanilla LLVM-main):
```
sollve_vv/tests/4.5/taskloop/test_taskloop_num_tasks.c:67:4: error: expected expression
   67 |    else
      |    ^
```
When enclosing `OMPVV_INFOMSG(...)` with braces: `{ OMPVV_INFOMSG(...) }` it compiles and passes.
If you don't mind could you look into that?
Otherwise, please let me know and I'll investigate further from our end.

Reply:
You are right, I just created a PR #803 to solve the problem. Thanks